app/views/application/equality-monitoring/sex.html

```diff
@@ -9,21 +9,32 @@
 {% endblock %}
 
 {% block primary %}
-  {{ govukRadios({
-    items: [{
-      text: "Female"
-    }, {
-      text: "Male"
-    }, {
-      text: "Intersex"
-    }, {
-      divider: "or"
-    }, {
-      text: "Prefer not to say"
-    }]
-  } | decorateApplicationAttributes(["equality-monitoring", "sex"])) }}
 
-  {{ govukButton({
-    text: "Continue"
-  }) }}
+  <p class="govuk-body">Training providers will not see your answer to this question when deciding whether to offer you a place.</p>
+
+  <p class="govuk-body">They will only see your answer if you accept an offer from them.</p>
```

Comment:
Should this also say that it'll be used for their teacher record? It'll go in to the provider's SSR and in Register / DQT

Reply:
I think "the provider’s SSR" is covered by the phase listed. Not sure if we need to explicitly mention Register and DQT though...

Reply:
We'll have to come back to this I think. Possibly also need to do a review of the privacy policy (and actual policy?).